dashboard/src/main/webapp/package.json

```diff
@@ -32,6 +32,7 @@
     "core-js": "^3.8.3",
     "element-plus": "^2.3.6",
     "moment": "^2.29.4",
+    "pinia": "^2.1.7",
```

Comment:
I just learned about pinia. Can the data in the store be shared between tabs of a browser?

Reply:
I suggest remove `pinia`,  we can just use `localStorage`.  @yl09099 @zuston @rickyma PTAL

Reply:
> I suggest remove `pinia`, we can just use `localStorage`. @yl09099 @zuston @rickyma PTAL

In this case, a global store is required to monitor the address change of the entire page, and localStorage cannot be used, which cannot monitor the change of the requested address.